examples/multiplatform-example/dokkatoo/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  kotlin("multiplatform") version "1.8.22"
+  kotlin("multiplatform") version "1.9.0"
```

Comment:
Updated because earlier KGP (specifically multiplatform) versions did not support configuration cache

add 'minimum Kotlin version is 1.9.0' to README